### ISSUE TITLE
completion for function

### ISSUE THREAD

**Issue by edmBernard** [google/python-fire#14]
The completion on function is missing

```
def add(self, first1, second1=123):
    print({"first1": first1, "second1": second1})
    return first1 + second1

def main():
    fire.Fire(add)

if __name__ == '__main__':
    main()
```

the completion file : 
```
# DO NOT EDIT.
# This script is autogenerated by fire/completion.py.

_complete-test_firepy()
{
  local start cur opts
  COMPREPLY=()
  start="${COMP_WORDS[@]:0:COMP_CWORD}"
  cur="${COMP_WORDS[COMP_CWORD]}"

  opts=""



  COMPREPLY=( $(compgen -W "${opts}" -- ${cur}) )
  return 0
}

complete -F _complete-test_firepy ./test_fire.py
```

**Comment by dbieber** [comment]
Thank you for finding and reporting this issue.

**Comment by dbieber** [comment]
[By the way, you probably don't want the argument 'self' in your add function.]

**Comment by edmBernard** [comment]
yeah that because for example I just change a bit my previous example : https://github.com/google/python-fire/issues/11 :)

**Comment by dbieber** [comment]
Resolved by 9eb788624958453053c96e37a16521f87b7651bd.
